packages/grafana-ui/src/components/Input/Input.tsx

```diff
@@ -59,6 +73,7 @@ export const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
           ref={ref}
           className={styles.input}
           {...restProps}
+          disabled={readOnly}
```

Comment:
This component looks to be spreading `restProps` which should mean we can already pass `disabled` to this input component.

Reply:
It should yes, but the "disabled" props is always `undefined` no matter what the value is.
I suspect the forwardRef to interact here

Reply:
fixed